app/holds/add-hold.js

```diff
@@ -4,7 +4,7 @@ const { sendHoldEvent } = require('../event')
 
 const addHold = async (frn, holdCategoryId, transaction) => {
   const hold = await db.hold.create({ frn, holdCategoryId, added: Date.now() }, { transaction })
-  await sendHoldEvent(hold, ADDED)
+  await sendHoldEvent(hold.get({ plain: true }), ADDED)
```

Comment:
Raw is not supported in `create` function, so must extract plain object from sequelize model to ensure only hold data published as event.